### ISSUE TITLE
unhandled exception trying to balance an account

### ISSUE THREAD

**Issue by andreva7** [MoneyTools/MyMoney.Net#133]
System.ArgumentOutOfRangeException: The added or subtracted value results in an un-representable DateTime. (Parameter 'value')
   at System.DateTime.ThrowDateArithmetic(Int32 param)
   at System.DateTime.AddTicks(Int64 value)
   at Walkabout.Views.Controls.BalanceControl.GetNextStatementDate(DateTime date)
   at Walkabout.Views.Controls.BalanceControl.Reconcile(MyMoney money, Account a, StatementManager statementManager)
   at Walkabout.MainWindow.BalanceAccount(Account a)
   at Walkabout.MainWindow.OnAccountsPanelBalanceAccount(Object sender, ChangeEventArgs e)
   at Walkabout.Views.Controls.AccountsControl.OnBalanceAccount(Object sender, ExecutedRoutedEventArgs e)
   at System.Windows.Input.CommandBinding.OnExecuted(Object sender, ExecutedRoutedEventArgs e)
   at System.Windows.Input.CommandManager.FindCommandBinding(CommandBindingCollection commandBindings, Object sender, RoutedEventArgs e, ICommand command, Boolean execute)
   at System.Windows.Input.CommandManager.FindCommandBinding(Object sender, RoutedEventArgs e, ICommand command, Boolean execute)
   at System.Windows.Input.CommandManager.OnExecuted(Object sender, ExecutedRoutedEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.Input.RoutedCommand.ExecuteImpl(Object parameter, IInputElement target, Boolean userInitiated)
   at MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

**Comment by andreva7** [comment]
There was a unique operation involved here:

I had an account for my lowes CC. I added a transaction and balanced the account.
I realized I had a duplicate account for this in mymoney. So I did a copy\paste of the transaction from the balanced account to the other account.
I delete the balanced lowes account
Now I am trying to balance the "other" lowes account (which only has this one transaction) and I get this exception
